client/context/UserContext.jsx

```diff
@@ -13,7 +13,7 @@ export const useUserUpdate = () => (
 );
 
 export const UserProvider = ({ children }) => {
-  const [User, setUser] = useState(null);
+  const [User, setUser] = useState(3);
```

Comment:
I don't think this change should be committed since it was probably hardcoded to test features out.

Reply:
OOPS! Will change this now